Docker instruction

```diff
@@ -0,0 +1,129 @@
+# Pull this files 
+docker pull dimitrii89/headnode:headnode
+docker pull dimitrii89/worker:worker
+
+# Open the ports: 8088, 9870
+
+# Dockerfiles:
+
+# Dockerfile_hn
+
+FROM centos:7
+
+MAINTAINER Dima <dimitry.vlasov@gmail.com>
+
+ARG HADOOP_HOME="/usr/local/hadoop/current"
+ENV HADOOP_HOME="/usr/local/hadoop/current"
```

Comment:
Why define that 2 times?

Reply:
ARG variables can be used to set default settings for ENV variables from the command line during the image build process.
That's why I wrote these lines. And it works. I didn't find any mistakes when creating images and then checking the browser.

Reply:
But there is no connections between `ARG` and `ENV`.
If you want to set `ENV` from arg then:
```
ARG HADOOP_HOME="/usr/local/hadoop/current"
ENV HADOOP_HOME="$HADOOP_HOME"
```